frontend/lib/json/event.dart

```diff
@@ -1,54 +1,27 @@
-// class EventInfo {
```

Comment:
Remove unused code

Reply:
Looks good to me. Frontend and backend now are consistent